### PR TITLE
task(readme): Certificate installation instructions on iOS and Android

### PR DESCRIPTION
**Pull request by ryanbas21**
should describe the steps to generate a root certificate and install a
signed certificate on the local server

https://bugster.forgerock.org/jira/browse/SDKS-1213

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,19 +36,131 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
...
+
+To modify the client portion of this project, you'll need to be familiar with the following React patterns:
+
+1. [Functional components and composition](https://reactjs.org/docs/components-and-props.html)
+2. [Hooks (including custom hooks)](https://reactjs.org/docs/hooks-intro.html)
+3. [Context API](https://reactjs.org/docs/hooks-reference.html#usecontext)
+4. [React Navigation](https://reactnavigation.org/)
+5. [React Native](https://reactnative.dev/)
+
+#### Styling and CSS
```
Comment (cerebrl) [inline README.md:140]: Should probably remove CSS as I believe it's not relevant for mobile.

File: README.md [hunk 2]
```diff
@@ -16,19 +36,131 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
...
+```sh
+# In one terminal window, run the following watch command
+# This "watches" the client source files for changes and rebuilds when needed
+$ npm run watch
+
+# In another terminal window, run the dev servers for both client and server
+$ npm run servers
+```
+
+Now, you should be able to visit `https://react.example.com:8443`, which is your web app or client (the Relying Party in OAuth terms). This client will make requests to your AM instance, (the Authorization Server in OAuth terms), which will be running on whatever domain you set, and `https://api.example.com:9443` as the REST API for your todos (the Resource Server). Enjoy!
```
Comment (cerebrl) [inline README.md:122]: This should be modified to suit React Native.

File: README.md [hunk 3]
```diff
@@ -16,19 +36,131 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
...
+
+## Learn About Integration Touchpoints
+
+This project contains comments describing the SDK integration points. We utilize both, the `iOS` bridge code and the Javascript SDK. Please see the comments in the source code to follow along with the associated blog post.
```
Comment (cerebrl) [inline README.md:126]: Please remove this.

File: README.md [hunk 4]
```diff
@@ -16,19 +16,22 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
...
 
-    ```
-      [!] CocoaPods could not find compatible versions for pod "FRAuth":
-    ```
-
-    run `pod repo update` and then `pod install`
+## General Troubleshooting
 
+1. Make sure `libFRAuth.a` is added to your Target's "Frameworks, Libraries, and Embedded Content".
+2. Make sure you have a `.env` file with valid values.
```
[the remark this answers is not in the document]
Comment (cerebrl) [reply README.md:159]: This should be changed to `.env.js`.

File: README.md [hunk 5]
```diff
@@ -16,19 +36,131 @@ ForgeRock protected native/hybrid application using React Native and the ForgeRo
...
+REST_OAUTH_SECRET=secret (the secret for the private OAuth 2.0 client/application)
+WEB_OAUTH_CLIENT=example-react-app (the name of the public OAuth 2.0 client/application)
+```
+
+### Installing Dependencies and Run Build
+
+```sh
+# Install the project dependencies
+$ npm install
+```
```
Comment (cerebrl) [inline README.md:90]: We should mention `pod install` as well

**Review by cerebrl — CHANGES_REQUESTED** [review]

File: docs/ios-certificates.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Create a signed certificate
+
+- In a terminal, navigate to your application directory and ensure that `domainScript.sh` has run access.
+  `$ chmod -775 ./domainScript.sh`
+  - run `./domainScript.sh [your-domain-name]`
+  - It is important that [your-domain-name] is the domain where you plan to install the certificate. For example, the local server for this application could use `api.example.com` - which includes the subdomain.
+  - Fill out the form, including the password. This generates a few new files with the following format:
+    [your-domain-name].(key,crt,csr,ext)
+  - Add the generated certificate files as the serving SSL certificate in your server. For this app, populate the .env file with the appropriate values
```
[the remark this answers is not in the document]
Comment (cerebrl) [reply docs/ios-certificates.md:66]: We should updated this to reflect the new `.env.js` file, yeah?

**Review by cerebrl — CHANGES_REQUESTED** [review]

**Review by cerebrl — APPROVED** [review]

**Review by spetrov — APPROVED** [review]
Looks good to me. 👍🏻 
Pretty straightforward and easy to follow.

**Pull request merged**